File: Front/src/app/models/users/user-data.ts
import { UserBase } from './user-base';

export class UserData extends UserBase {
	public ID: string;
	public FullName: string;
	public CompanyCode: string;
	public CompanyName: string;
	public CreatorUserID: string;
	public CreatorUserName: string;
	public ModifierUserID: string;
	public ModifierUserName: string;

	public LastLogin?: Date;
	public CreatedDate?: Date;
	public UpdatedDate?: Date;
	public ApiKey:string;
}

File: Back/BancaNet.Backend.Framework/Models/Users/UserData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BancaNet.Backend.Framework.Models.Users
{
	public sealed class UserData : UserBase
	{
		public string ID { get; set; }
		public string ApiKey { get; set; }
		public string FullName { get; set; }
		public string CompanyName { get; set; }
		public string CreatorUserID { get; set; }
		public string CreatorUserName { get; set; }
		public string ModifierUserID { get; set; }
		public string ModifierUserName { get; set; }
		public DateTime? CreatedDate { get; set; }
		public DateTime? UpdatedDate { get; set; }
		public DateTime? LastLogin { get; set; }
	}
}
